python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -141,6 +148,7 @@ def __init__(
             initial_state=initial_state,
             override=override,
             simulation_mode=simulation_mode,
+            extra_commands={self.do_lockMotion, self.do_unlockMotion},
```

Comment:
This should be the name of the commands as strings, not the methods, e.g.; 

```
extra_commands={"lockMotion", "unlockMotion"}
```

Reply:
Fixed in the commit: [b4e39d1](https://github.com/lsst-ts/ts_mtrotator/pull/97/commits/b4e39d11dc643201ed9fc28fd9b89b0c45c65069). Thanks!